packages/web/src/components/nav/desktop/AccountSwitcher/AccountListContent.tsx

```diff
@@ -0,0 +1,103 @@
+import { useCallback } from 'react'
+
+import { ID, ManagedUserMetadata, UserMetadata } from '@audius/common/models'
+import { Box, Flex, IconUserArrowRotate, Text } from '@audius/harmony'
+import styled from '@emotion/styled'
+
+import { AccountSwitcherRow } from './AccountSwitcherRow'
+
+const messages = {
+  switchAccount: 'Switch Account',
+  maangedAccounts: 'Managed Accounts'
+}
+
+export type AccountListContentProps = {
+  accounts: ManagedUserMetadata[]
+  managerAccount: UserMetadata
+  currentUserId: ID
+  onAccountSelected: (user: UserMetadata) => void
+}
+
+const StyledList = styled.ul`
+  all: unset;
+  list-style: none;
+`
+
+export const AccountListContent = ({
+  accounts,
+  managerAccount,
+  currentUserId,
+  onAccountSelected
+}: AccountListContentProps) => {
+  const onUserSelected = useCallback(
+    (user: UserMetadata) => {
+      if (user.user_id !== currentUserId) {
+        onAccountSelected(user)
+      }
+    },
+    [currentUserId, onAccountSelected]
+  )
+  return (
+    <Flex
+      direction='column'
+      w={360}
+      borderRadius='s'
+      border='strong'
+      backgroundColor='white'
+    >
+      <Flex
+        backgroundColor='white'
+        borderBottom='default'
+        alignItems='center'
+        justifyContent='flex-start'
+        ph='l'
+        pv='s'
+        gap='s'
+      >
+        <IconUserArrowRotate size='s' color='default' />
+        <Text variant='title' size='m' color='default'>
+          {messages.switchAccount}
+        </Text>
+      </Flex>
+      <StyledList role='menu' tabIndex={-1}>
+        <li
+          role='menuitem'
+          tabIndex={0}
+          onClick={() => onUserSelected(managerAccount)}
+        >
+          <AccountSwitcherRow
+            user={managerAccount}
+            isSelected={currentUserId === managerAccount.user_id}
+          />
+        </li>
+        <Box
+          ph='l'
+          pv='s'
+          backgroundColor='surface1'
+          borderTop='default'
+          borderBottom='default'
+        >
+          <Text variant='title' size='xs' color='default'>
+            {messages.maangedAccounts}
+          </Text>
+        </Box>
+
+        {accounts.map(({ user }, i) => (
+          <li
+            key={user.user_id}
+            role='menuitem'
+            onClick={() => onUserSelected(user)}
+            tabIndex={-1}
```

Comment:
-1 kinda magic, can it be a null or undefined?

Reply:
I don't think so?
Negative numbers have a specific meaning for tabIndex:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
TIL!